tests/unit/test_dataset_builders/test_contents_define_vlm_dataset_builder.py

```diff
@@ -80,4 +82,6 @@ def test_contents_define_vlm_dataset_builder(
     expected_df = pd.DataFrame.from_dict(expected)
     expected_df.sort_index(axis=1, inplace=True)
     result.sort_index(axis=1, inplace=True)
+    assert "define_vlm_mandatory" in result
+    assert result.define_vlm_mandatory[0] == 'Yes'
     assert result.equals(expected_df)
```

Comment:
Sorry, @RamilCDISC . When I looked at this over the weekend, I didn't see that line 87 is already checking the whole result against expected. So this means that lines 85 and 86 are not needed at all now that you have lines 26 and 49.

However, lines 26 and 49 should be updated because currently they are single values instead of arrays. Here is a summary of changes that should be made:

line 26: Change to: `"define_vlm_mandatory": ["Yes", "No"],`
line 49: Change to: `"define_vlm_mandatory": ["Yes"],`
line 85: delete
line 86: delete